data/initrd/initrd.file_list

```diff
@@ -569,6 +570,10 @@ x etc/syslog.conf etc
 
 # additional scripts for linuxrc
 x scripts /
+# hmc exists only on s390
```

Comment:
Ok, so if there's any chance to see this menu entry ever also on ppc, this might be an argument for the long menu name.